src/cirrus/configuration.py

```diff
@@ -150,8 +158,14 @@ def load_configuration(package_dir=None):
         dirname = package_dir
 
     config_path = os.path.join(dirname, 'cirrus.conf')
+    if not os.path.exists(config_path):
+        repo_dir = _repo_directory()
+        config_path = os.path.join(repo_dir, 'cirrus.conf')
 
     if not os.path.exists(config_path):
+        config_path = None
+
+    if config_path is None:
```

Comment:
You could refactor to raise the `RuntimeError` where you set `config_path = None`, line 166